build/helpers/coverage.js

```diff
@@ -41,7 +73,34 @@ process.on("exit", () => {
         continue;
       }
 
-      parseScript(scriptCoverage);
+      if (scriptCoverage.url !== impl) {
+        continue;
+      }
+      const script = parseScript(scriptCoverage);
+
+      if (script === null) {
+        continue;
+      }
+
+      let sortedBlocks = script.coverage.sort(function(a, b) {
+        if (a.points < b.points) {
+          return 1;
+        }
+
+        if (a.points > b.points) {
+          return -1;
+        }
+
+        return 0;
+      });
```

Comment:
`Array.sort()` modifies the array in-place so there's no need to assign to another variable. The comparator can also be simplified:

```ts
script.coverage.sort((a, b) => a.points - b.points);
```